### PR TITLE
PR according to Issue

### PR DESCRIPTION
**Pull request by msseibel**
This PR was discussed here.

https://github.com/autoreject/autoreject/issues/196

### CODE REVIEW

**Comment by jasmainak** [comment]
can you fix the `flake8` issue?

File: autoreject/autoreject.py [hunk 1]
```diff
@@ -215,8 +215,10 @@ def get_rejection_threshold(epochs, decim=1, random_state=None,
     if decim > 1:
         epochs = epochs.copy()
         epochs.decimate(decim=decim)
-
-    cv = KFold(n_splits=cv, random_state=random_state)
+    
+    if isinstance(cv, int):
+        cv = KFold(n_splits=cv, random_state=random_state)
```
Comment (jasmainak) [inline autoreject/autoreject.py:220]: maybe use this: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.check_cv.html to check that the `cv` instance makes sense?

And then you need to update the docstring above to say you accept any `cv` object from `scikit-learn`

**Review by jasmainak — APPROVED** [review]
oops sorry I didn't see you had addressed the comments. Looks good to me. 

@agramfort do you want to merge?

**Comment by agramfort** [comment]
thx @msseibel

**Pull request merged**